apps/backend/src/app/modules/admin/admin.controller.ts

```diff
@@ -63,9 +67,36 @@ export class AdminController {
     private readonly mapsService: MapsService,
     private readonly mapReviewService: MapReviewService,
     private readonly usersService: UsersService,
-    private readonly adminActivityService: AdminActivityService
+    private readonly adminActivityService: AdminActivityService,
+    private readonly killswitchService: KillswitchService
   ) {}
 
+  @Patch('/killswitch')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @ApiOperation({ summary: 'Update active killswitches' })
+  @ApiNoContentResponse({
+    description: 'The switches were updated successfully'
+  })
+  @ApiBadRequestResponse({ description: 'Invalid switches type' })
+  @ApiBody({
+    type: Object,
+    description: 'Updated switches',
+    required: true
+  })
+  updateKillSwitch(@Body() switches: Killswitches): void {
+    this.killswitchService.updateKillswitches(switches).then((r) => r);
+  }
+
+  @Get('/killswitch')
+  @ApiOperation({ summary: 'Gets currently stored killswitches' })
+  @ApiNoContentResponse({
+    description: 'Found switches'
+  })
+  @ApiBadRequestResponse({ description: 'Invalid switches type' })
+  getKillSwitches(): object {
```

Comment:
Can't you use `Killswitches` instead of `object` here? And same thing for the ApiNoContentResponse as the ApiBody above, just a text-based type in the description in fine.